src/key_io.h

```diff
@@ -13,6 +13,8 @@
 
 #include <string>
 
+const std::size_t WITNESS_PROG_MAX_LEN = 40;
```

Comment:
nit if you retouch
```suggestion
constexpr std::size_t WITNESS_PROG_MAX_LEN = 40;
```

Reply:
Good point, I moved it inside key_io.cpp